### PR TITLE
#22: V7.0 Chapter 2 Test fixes

### PR DESCRIPTION
**Pull request by kamonson**
Chapter 2 Fixes From issue #22

### CODE REVIEW

**Comment by MarkMichaelis** [comment]
It appears this has already been merged.

File: src/Chapter02/Chapter02.csproj [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <StartupObject>AddisonWesley.Michaelis.EssentialCSharp.Shared.Program</StartupObject> 
   </PropertyGroup>
   <Import Project="..\Versioning.targets" />
+  <ItemGroup>
```
Comment (COsborn2) [inline src/Chapter02/Chapter02.csproj]: We might want to remove this Listing.. for now Im going to let it compile..but the next commit might see this file being removed
```suggestion
```

File: src/Chapter02/Chapter02.csproj [hunk 2]
```diff
@@ -6,6 +6,9 @@
     <StartupObject>AddisonWesley.Michaelis.EssentialCSharp.Shared.Program</StartupObject> 
   </PropertyGroup>
   <Import Project="..\Versioning.targets" />
+  <ItemGroup>
+    <Compile Remove="Listing02.01-2.06.SpecifyingLiteralValues.cs" />
```
Comment (COsborn2) [inline src/Chapter02/Chapter02.csproj]:
```suggestion
```

File: src/Chapter02/Chapter02.csproj [hunk 3]
```diff
@@ -6,6 +6,9 @@
     <StartupObject>AddisonWesley.Michaelis.EssentialCSharp.Shared.Program</StartupObject> 
   </PropertyGroup>
   <Import Project="..\Versioning.targets" />
+  <ItemGroup>
+    <Compile Remove="Listing02.01-2.06.SpecifyingLiteralValues.cs" />
+  </ItemGroup>
```
Comment (COsborn2) [inline src/Chapter02/Chapter02.csproj]:
```suggestion
```

**Comment by COsborn2** [comment]
Changes incorporated

**Pull request closed without merging**